workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/IReaderInternal.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Improbable.Worker;
+
+namespace Improbable.Gdk.Core.MonoBehaviours
+{
+    /// <summary>
+    ///     The interface that is used to signal a reader that a message has been received from the SpatialOS Dispatcher
+    ///     about the component that the reader is connected to.
+    /// </summary>
+    internal interface IReaderInternal
+    {
+        void OnAuthorityChange(Authority authority);
+        void OnComponentUpdate();
```

Comment:
Heads up: This interface here might be subject to change depending on the discussion in https://github.com/spatialos/UnityGDK/pull/121